woocommerce/woocommerce
# Remove unnecessary wooCommerce admin files

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/32360.

Removed unnecessary wooCommerce admin files since core has the same features.

I still keep our i18 languages files since we haven't updated i18n text domain (https://github.com/woocommerce/woocommerce/issues/32175). 

### How to test the changes in this Pull Request:

1. Review "Files changed" to see if it's okay to delete those files. (You can use the left sidebar to help you review this!)
2. Go to [this branch page](https://github.com/woocommerce/woocommerce/tree/remove/32360-unnecessary-wca-files/plugins/woocommerce-admin) and see if there are still some files we should remove 
3. Should run `pnpm nx clean woocommerce-admin` & `pnpm nx build woocommerce` successfully

### Other information:




### Changelog entry

Dev: remove unnecessary wooCommerce admin files

### FOR PR REVIEWER ONLY:
